.golangci.yml

```diff
@@ -18,7 +18,11 @@ issues:
 
 linters-settings:
   govet:
-    enable=fieldalignment: true
+    enable-all: true
+    disable:
+      - fieldalignment
```

Comment:
@camilamacedo86 as configuration was wrong, it was not enforcing `fieldalignment` rule

I disabled it because there are several files affected

I suggest we merge this one, and fix the linter errors in follow up